bplan/assets/html/app/components/map/popup.html

```diff
@@ -43,16 +43,14 @@
         </div>
     </div>
     <div class="row">
-        <div class="popup-entry">
-            <div data-ng-if="bplan.ausleg_www" class="col-xs-3 col-sm-3 col-md-3">
-                <div><a class="custom-btn btn-primary" target="_blank" href="{{ bplan.ausleg_www }}">Website</a></div>
-            </div>
-            <div data-ng-if="bplan.scan_www" class="col-xs-3 col-sm-3 col-md-3">
-                <div><a class="custom-btn btn-primary" target="_blank" href="{{ bplan.scan_www }}">zum B-Plan</a></div>
-            </div>
-            <div data-ng-if="bplan.grund_www" class="col-xs-3 col-sm-3 col-md-3">
-                <div><a class="custom-btn btn-primary" target="_blank" href="{{ bplan.grund_www }}">Begründung</a></div>
-            </div>
+        <div data-ng-if="bplan.ausleg_www" class="col-3">
```

Comment:
I think we are missing a margin 20px for top and bottom here (came with the popu-entry before)

Also the `col-3` causes the "zum B-Plan" button to wrap the text too quickly, we might need something bigger

Reply:
couldn't replicate but changed it so it's like the list item, as it was 3-col before i imagine the issue exists on prod too?! if still a problem hit me up